mobile/src/main/java/rs/readahead/washington/mobile/views/settings/MainSettings.kt

```diff
@@ -40,6 +50,16 @@ class MainSettings : Fragment() {
         view.findViewById<View>(R.id.about_n_help_settings_button).setOnClickListener {
             Navigation.findNavController(view).navigate(R.id.action_main_settings_to_about_n_help_settings)
         }
-        return view
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        if ((activity as OnFragmentSelected?)?.isCamouflage() == true) {
+            if (cm.isDefaultLauncherActivityAlias) {
+                Navigation.findNavController(view).navigate(R.id.action_main_settings_to_hide_tella)
```

Comment:
We already have `nav()` inside the `BaseFragment` so no need for `Navigation.findNavController(view)`